index.js

```diff
@@ -0,0 +1,76 @@
+const uuid = require('node-uuid');
+
+/**
+ * Filter for response messages
+ * @param  {object}  msg - message to check
+ * @return {Boolean}
+ */
+function isChildMessage(msg) {
```

Comment:
I'm beginning to think this should be `isChildMessage(parentMessage, message)`, which we would use in an RxJS filter like so:

```
const isChildMessage = enchannel.isChildMessage.bind(null, msg);
const childMessages = stream.filter(isChildMessage)
```